### ISSUE TITLE
Putting together a Guest Facilitator guide

### ISSUE THREAD

**Issue by abbycabs** [MozillaFoundation/mentorship-program#49]
With all the guest speakers we have on the cohort calls, I'm realizing we need to put together a guide for them!

Past guest speakers, what would be helpful?
Cohort participants, anything you think our guest speakers should know?

Anyone have an existing guide we can remix? I'd also love help putting this together if anyone is interested :)

----

Using the template
* make a copy of the slides
* edit the 'about the presenter' slide
* make sure you're comfortable sharing these slides
* anything missing or unclear? Remix the slides and make them your own!

Video call presenting. Make sure you have:
* a good internet connection
* a quiet place to call in
* make sure your camera works (not required)

Prep for presenting
* read through the full lesson
* watch this recording or someone else presenting the same slides (find good examples!)
* read through the assignment

... anything else to add?

**Comment by abbycabs** [comment]
Link to zoom sharing screen info that @bunnybooboo shared!

**Comment by chadsansing** [comment]
cc @chadsansing

**Comment by bunnybooboo** [comment]
Connect 10-15 mins before the call to make sure everything works.
Time yourself during a few dummy runs.
Stick a picture of a person next to your computer. It helps to remember you're connecting to real people.

**Comment by abbycabs** [comment]
Tagging all recent guest facilitators to see if they have any additional tips to share here for us to collate!

@annakrystalli
@anubhamane
@btayeg
@dbouquin
@dasaderi
@daniellecrobinson
@dbild
@bunnybooboo
@GrantRVD
@Zebetus
@jasonclark
@KadeMorton
@lequanne
@marcosvital
@MinnSoe
@philippbayer
@aariops
@cameralibre
@kitsunenosarat
@spMohanty
@psubhashish

Also tagging Hosts, since they have a lot of experience on these calls
@chartgerink
@macruzbar
@Monsauce
@priyankanag
@rainsworth
@schae234
@syswsi
@yochannah

**Comment by philippbayer** [comment]
Could we collect which systems the Zoom screen sharing works on? On my Fedora 27 it doesn't work at all, I think most Linux systems have issues there

**Comment by annakrystalli** [comment]
All I'd add is to read through the full lesson before remixing slides for some inspiration and feel free to add relevant detail. Also consider remixing in some links to your favourite examples of whatever you're presenting.

**Comment by chartgerink** [comment]
Some affirmations? I always get nervous and the people are so relaxed that it always works out a-okay :)

**Comment by macruzbar-zz** [comment]
When presenting:
- Remember the majority of your audience speaks English as a second language. Speak slow and be clear. 
- Try not to mix external topics, it could be confusing
- Connect the concepts and ideas with real life examples and applications.

**Comment by cameralibre** [comment]
Note: When presenting from Google Slides, it opens a new window for the presentation. When you share screens in Zoom, you choose a **single browser window** to share  - this means that if you have external links that you want to show in the browser while you are presenting, you either need to:
1. [within Zoom] Change the shared screen to another browser window and then back to your presentation when you're done - this can be a bit slow and awkward.
or 
2. [within Google Slides] embed screenshots of the external content that you want to show _within the presentation itself_ with a link that viewers can follow later. This works well unless you want to show dynamic content, or demo an action.
or 
3. stick to a single browser window, and present your slides from the Editor view of Google slides, it's not as pretty, but it allows you to move between different tabs, demo stuff and come back, all at a natural speed.

**Comment by cameralibre** [comment]
@philippbayer it works fine on Solus, and I remember Ubuntu working too, but I haven't checked for a while.

**Comment by kitsunenosaraT** [comment]
I think @macruzbar makes excellent points, especially about the English as second language part.
I would add:
- don't use too much text in the slides, just highlight the key points to anchor attention
- along the slides, share a script/notes for people struggling with spoken English.
- make your presentation more interactive by inviting people to share tips and thoughts in the etherpad.

**Comment by abbycabs** [comment]
Loving this thread - these are excellent tips! Keep them coming :)

I'll point new guest facilitators to this thread while the guide is pending

**Comment by marcosvital** [comment]
Hi! Little late here, sorry...

@macruzbar First point also works the other way around, to relax those (like me!) that are a little insecure about their own spoken english:  *the majority of your audience speaks English as a second language*, so they won't mind your accent and will understand your struggle when trying to find the right words.  ;)

**Comment by psubhashish** [comment]
(not in any particular order)
- If you can cover all the topics that you are supposed to cover, do wear a smiling face, and add some good humor.
- Check if your computer's camera is facing towards a light. It makes you look like a silhouette (unless you want that effect!). How someone on the other side sees you totally affect the way they learn. 
- Keep some time right after the call for any Etherpad/Twitter discussions. They are like those great discussions that happen off stage in real conferences.
- Share some relevant quick tips that helped you personally. People love to connect that (human) way.

**Comment by abbycabs** [comment]
Some tips from @rhoadsstevens. There are great points here on bringing in your personal perspective and paying attention to the flow of the slides & context with previous presentations.

*  I really made sure I understood how the slides built off each other before I made changes. Then, after I made changes, I also made sure the slides continued to build off each other.
* I felt it would be good to bring up a bit more about why I, personally, still consult the Open Leadership Framework and find it meaningful. 
* I also wanted to reference the project I had worked on in Open Leadership.
